sdk/python/feast/infra/contrib/spark_kafka_processor.py

```diff
@@ -0,0 +1,139 @@
+from types import MethodType
+from typing import List
+
+from pyspark.sql import DataFrame, SparkSession
+from pyspark.sql.avro.functions import from_avro
+from pyspark.sql.functions import col, from_json
+
+from feast.data_format import AvroFormat, JsonFormat
+from feast.data_source import KafkaSource
+from feast.infra.contrib.stream_processor import (
+    ProcessorConfig,
+    StreamProcessor,
+    StreamTable,
+)
+from feast.stream_feature_view import StreamFeatureView
+
+
+class SparkProcessorConfig(ProcessorConfig):
+    spark_session: SparkSession
+
+
+class SparkKafkaProcessor(StreamProcessor):
+    # TODO: wrap spark data in some kind of config
```

Comment:
remove these comments the processor config exists already

Reply:
done